docs/DeveloperGuide.md

```diff
@@ -143,8 +146,26 @@ The `Option` and `OptionList` classes implements the `Displayable` and `Displaya
 The `Option` object stores one option of a question while the `OptionList` object stores all 4 options of the same 
 question. The class diagram below illustrates the structure of both classes. 
 
+The `Option` class implements the `Displayable` interface while the `OptionList` class implements the `DisplayableList` interface. The illustration below shows the class diagram of `Option` and `OptionList`. 
+
 ![Option_and_OptionList_Class](./images/Option.png)
 
+The `Option` object stores the description of one option from a question. It also indicates if the option is the correct answer for the question by using the `isCorrectAnswer` boolean. 
+The `OptionList` object stores all 4 options of the same question. 
+
+#### 2.2.5. Implementation of Option and OptionList
+
+An example of when the `Option` and `OptionList` classes are used is when the quiz mode is activated. The quiz requires all the options of a particular question to be printed out. 
+
+![Option_and_OptionList_Sequence](./images/OptionSequence.png)
```

Comment:
I think the method calls a bit wrong as you should point to the object that is called. Liked question.getInnerList() should call on a question object and getInnerList() should call on a questionList object